CHANGELOG.next.asciidoc

```diff
@@ -309,7 +309,7 @@ otherwise no tag is added. {issue}42208[42208] {pull}42403[42403]
 - The environment variable `BEATS_ADD_CLOUD_METADATA_PROVIDERS` overrides configured/default `add_cloud_metadata` providers {pull}38669[38669]
 - When running under Elastic-Agent Kafka output allows dynamic topic in `topic` field {pull}40415[40415]
 - The script processor has a new configuration option that only uses the cached javascript sessions and prevents the creation of new javascript sessions.
-- Update to Go 1.24.3. {pull}44270[44270]
+- Update to Go 1.24.4. {pull}44696[44696]
```

Comment:
As per documentation team, the changelogs should now be in release.d
[Reference](https://github.com/elastic/beats/pull/44576#discussion_r2137074745)

Reply:
Thanks, we still need this file for 8.x so it has to go in both places until we can abstract it away with new tooling.